lib/db.js

```diff
@@ -204,11 +204,16 @@ DB.prototype.get = function(domain, req) {
         return this._getSchema(tableName)
         .then(function(schema) {
             self.schemaCache[tableName] = schema;
-            return self._get(tableName, req, schema, true);
+            return self._get(tableName, req, schema, {
+                includePreparedForDelete: true
+            });
         });
     } else {
         return P.try(function() {
-            return self._get(tableName, req, self.schemaCache[tableName], true);
+            return self._get(tableName, req, self.schemaCache[tableName], {
+                includePreparedForDelete: true,
+                withTTLs: req.withTTL
```

Comment:
I think it'd be good to keep consistent, so I'd lose the `s` at the end of the option field to match `req.withTTL`